templates/boltzmann/middleware/log.ts

```diff
@@ -21,6 +22,31 @@ function log ({
   }
   bole.output({ level, stream })
 
+  void `{% if honeycomb %}`
+  honeycomb.logger = bole('boltzmann:honeycomb')
+  if (honeycomb.features.beeline) {
+    honeycomb.logger.warn(
```

Comment:
I'm split on this. On one hand I'd like *us* to get off beelines as soon as it makes sense, but on the other Honeycomb itself has *not* deprecated beeline, merely placing it in "maintenance mode". What do people think?